packages/document/docs/en/config/options/options.mdx

```diff
@@ -156,16 +150,38 @@ new WebDoctorWebpackPlugin({
 - Default: false
 - Description
 
-  开启或关闭需要 tos 上传， true 是关闭 tos 上传，false 是开启 tos 上传。
-
   Turn tos upload on or off. True is to turn off tos upload, and false is tos upload on.
 
 - Example
 
 ```js
 new WebDoctorWebpackPlugin({
-        disableClientServer: false, // 建议 CI 环境下关闭 server 能力，否则会卡住 pipeline 流程，检测 CI 环境可以使用 `process.env.CI_ACTOR`.
+        disableClientServer: false,  // It is recommended to disable the server capability in the CI environment, otherwise it will block the pipeline process. You can check the CI environment using `process.env.CI_ACTOR`.
         features: ['loader', 'bundle', 'plugins'],
         disableTOSUpload: true,
       }),
 ```
+
+### supports
+
+- **Type:** [Supports Types](#supports-types)
+- **Optional:** `true`
+- **Default:** `undefined`
+
+:::danger
```

Comment:
this tip should be moved to `supports.banner`

Reply:
fixed~